src/java/org/apache/cassandra/tools/LoaderOptions.java

```diff
@@ -494,24 +508,46 @@ public Builder parseArgs(String cmdArgs[])
                     System.exit(1);
                 }
 
-                if (cmd.hasOption(THROTTLE_MBITS))
+                if (cmd.hasOption(THROTTLE_MEGABITS) && cmd.hasOption(THROTTLE_MEBIBYTES))
+                {
+                    errorMsg(String.format("Both '%s' and '%s' were provided. Please only provide one of the two options", THROTTLE_MEGABITS, THROTTLE_MEBIBYTES), options);
+                }
+
+                if (cmd.hasOption(INTER_DC_THROTTLE_MEGABITS) && cmd.hasOption(INTER_DC_THROTTLE_MEBIBYTES))
+                {
+                    errorMsg(String.format("Both '%s' and '%s' were provided. Please only provide one of the two options", INTER_DC_THROTTLE_MEGABITS, INTER_DC_THROTTLE_MEBIBYTES), options);
+                }
+
+                if (cmd.hasOption(THROTTLE_MEGABITS))
+                {
+                    throttleMebibytes = DataRateSpec.IntMebibytesPerSecondBound.megabitsPerSecondInMebibytesPerSecond(Integer.parseInt(cmd.getOptionValue(THROTTLE_MEGABITS)))
+                                                                               .toMebibytesPerSecondAsInt();
```

Comment:
let me think about this. I have some ideas, but one of the options is to use a `double` type instead of `int`. Another idea is to keep two variables, one for mibs and one for MiBs. I will probably end up going with the solution that produces the more accurate result in the tests.

Reply:
As long as the user sees only ints and input match output - I am good :-) Probably both versions should work